src/main/java/sirius/biz/storage/layer3/VirtualFileSystem.java

```diff
@@ -120,4 +124,37 @@ public String makePath(String... parts) {
         // checks or cleanups...
         return "/" + Strings.join("/", parts);
     }
+
+    /**
+     * Returns the storage space associated with the given file.
+     *
+     * @param virtualFile the virtual file to obtain the storage space
+     * @return the {@link BasicBlobStorageSpace} sitting on top of the virtual file
+     */
+    public Optional<? extends BasicBlobStorageSpace<?, ?, ?>> fetchBlobStorageSpace(VirtualFile virtualFile) {
+        Optional<? extends BasicBlobStorageSpace<?, ?, ?>> blobStorageSpace =
+                virtualFile.tryAs(MongoBlobStorageSpace.class);
```

Comment:
Only use BlobStorageSpace here

Reply:
Do we need this as api anyway, this is just a boilerplate for another single method call then...